src/api/README.md

```diff
@@ -46,77 +46,83 @@ When a client gets rate limited the API responds with the `429 Too Many Requests
 
 ## Code examples
 
-### Terminal (curl)
+### Terminal cURL
 
 ``` bash
 curl --request POST \
   --url https://graphql.apirocket.io/ \
   --header 'Authorization: Bearer YOUR_API_KEY_TOKEN' \
   --header 'Content-Type: application/json' \
   --header 'Accept: application/json' \
-  --data '{"query":"COPY_AND_PASTE_YOUR_QUERY_HERE"}'
+  --data '{"query":"YOUR_QUERY_HERE"}'
 ```
 
-### Javascript fetch
+### Javascript Fetch
 
 ``` js
-const myQuery = `COPY_AND_PASTE_YOUR_QUERY_HERE`;
-
-fetch("https://graphql.apirocket.io", {
-	"method": "POST",
-	"headers": {
-		"Content-Type": "application/json",
-		"Authorization": "Bearer YOUR_API_KEY_TOKEN",
-	},
-	"body": JSON.stringify({query: myQuery})
-	})
-	.then(response => response.json())
-	.then(json => console.log(json.data))
-	.catch(err => { console.error(err); });
+// Set the main variables and your custom QUERY and KEY TOKEN.
+const url = 'https://graphql.apirocket.io';
+const method = 'POST';
+const headers = {
+	'Content-Type': 'application/json',
+	Authorization: 'Bearer YOUR_API_KEY_TOKEN',
+};
+const body = JSON.stringify({
+	query: `YOUR_QUERY_HERE`
+});
+
+fetch(url, { method, body })
+	.then((data) => data.json())
+	.then((response) => console.log(response))
+	.catch(err => console.error(err));
 ```
 
 ### Javascript Axios
 
 ``` js
+// Import the main dependencies.
 import axios from "axios";
-const myQuery = `COPY_AND_PASTE_YOUR_QUERY_HERE`;
-
-const options = {
-	method: 'POST',
-	url: 'https://graphql.apirocket.io/',
-	headers: {
-	"Content-Type": "application/json",
-	"Authorization": "Bearer YOUR_API_KEY_TOKEN"
-	},
-	data: JSON.stringify({query: myQuery})
+
+// Set the main variables and your custom QUERY and KEY TOKEN.
+const url = 'https://graphql.apirocket.io';
+const method = 'POST';
+const headers = {
+	'Content-Type': 'application/json',
+	Authorization: 'Bearer YOUR_API_KEY_TOKEN',
```

Comment:
@JoseJPR Authorization is without quotes?

Thanks

Reply:
Tools like EsLint will tell you that it is not necessary in these types of properties. But feel free, you can add, for example, a single quotes in all cases if you see it better.

Reply:
Hi @juanmiguelguerrero, any advance? We continue with this proposal?